src/network/networkd-link.c

```diff
@@ -2276,6 +2276,9 @@ static int link_enter_join_netdev(Link *link) {
 
         HASHMAP_FOREACH(netdev, link->network->stacked_netdevs, i) {
 
+                if (netdev->ifindex)
```

Comment:
please see CODING_STYLE: do not rely on C's downgrade-to-bool feature for anything that is actually a bool, or for ptrs. i.e. for comparing numeric values, do this explicitly with != 0 or so

Reply:
OK updated.